otherlibs/stdune/src/int.ml

```diff
@@ -12,7 +12,27 @@ include Comparable.Make (T)
 
 let equal (a : t) b = a = b
 
-let hash (t : t) = t
+(* this implementation (including the comment) is taken from base *)
```

Comment:
```suggestion
(* This implementation (including the comment) is taken from the Base library. *)
```

Reply:
Did this change get undone somehow?

Reply:
Which change got undone? This PR https://github.com/ocaml/dune/pull/8259 will not be merged because you asked to move all the hash stuff into one PR.

Reply:
Never mind. All good!